app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -3,41 +3,38 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:fillViewport="true">
 
     <android.support.constraint.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         tools:context="org.systers.mentorship.view.activities.SignUpActivity">
 
         <android.support.v7.widget.AppCompatTextView
-            android:id="@+id/textView"
+            android:id="@+id/signUpTextview"
```

Comment:
rename this ID to `tvSignUp` just to follow the rest of the xml elements ids current convention

Reply:
Fixed in the latest commit.